opsdroid/__main__.py

```diff
@@ -111,11 +112,25 @@ def print_example_config(ctx, param, value):
     ctx.exit(0)
 
 
-def edit_config(ctx, param, value):
+def edit_files(ctx, param, value):
     """Easy way to edit the config file."""
-    if not value or ctx.resilient_parsing:
+    if value == 'config':
+        file = DEFAULT_CONFIG_PATH
+    elif value == 'log':
+        file = DEFAULT_LOG_FILENAME
+    else:
         return
-    subprocess.run([os.environ.get('EDITOR', 'vi'), DEFAULT_CONFIG_PATH])
+
+    editor = os.environ.get('EDITOR', 'vi')
+    if editor == 'vi':
+        if click.prompt("You are about to open a file in vi/vim, "
```

Comment:
We could add a config option similar to `welcome-message` which allows you to disable this prompt.